src/app/tabs/admin/components/AdminTabComponent.js

```diff
@@ -120,6 +120,11 @@ class AdminTabComponent extends Component {
               <DangerZoneContainer />
             </Col>
           </Row>
+          <Row>
+            <Col>
+              <Link to="/newAdmin">Try the new admin!</Link>
+            </Col>
+          </Row>
```

Comment:
If we merge this into the `develop` branch, do we need to remove it when it gets published to production? Or are we planning on having this link from here on out?

Reply:
Let's leave it in prod. This does not make things more messy than they are right now 😆